framework/Runners/DistributedMemoryRunner.py

```diff
@@ -99,7 +84,7 @@ def isDone(self):
     if self.thread is None:
       return True
     else:
-      return self.thread.finished
+      return self.thread in ray.wait([self.thread], timeout=1e-10)[0]
```

Comment:
maybe make a variable for timeout for easy maintenance.

Reply:
done